SAMtemplates/common_lambda_resources.yml

```diff
@@ -108,6 +109,8 @@ Resources:
             Resource:
               - !GetAtt LambdaLogGroup.Arn
               - !Sub ${LambdaLogGroup.Arn}:log-stream:*
+              - !GetAtt LambdaLogGroup.Arn
+              - !Sub ${LambdaLogGroup.Arn}:log-stream:*
```

Comment:
doesnt this just duplicate the lines above?

Reply:
yeah your right, i think that must have slipped through during the merge conflicts